packages/suite-native-navigation-root/src/navigation/routes.ts

```diff
@@ -1,23 +1,26 @@
+import { NavigatorScreenParams } from '@react-navigation/native';
+import { HomeStackParamList } from '@suite-native/home';
+
 import { TabsOption } from '@suite-native/navigation';
 
 export enum RouteTabs {
-    Home = 'Home',
+    HomeStack = 'HomeStack',
```

Comment:
I just followed convention that was used in settings to keep it same. IMO it's not bad because it shows that it's not just simple route but has some nested routes as well.

Reply:
Sure, it does make sense! It's just that the enum is named Tabs, so it is clearly about the tabs at the bottom and not routes in general :) But whatever is more beneficial for the structure in your opinion!

Reply:
The way I look at it: 
1. Tabs enum, because the tabs of the app navigation are just these elements
2. Then inside the enum naming them Home / HomeStack to differentiate between flat / nested sub routes